expression/expression.go

```diff
@@ -172,10 +169,11 @@ func evalExprToString(expr Expression, row []types.Datum, _ *variable.StatementC
 	}
 	tc := expr.GetType().ToClass()
 	if tc == types.ClassString {
-		return val.GetString(), false, nil
+		// here we cannot use val.GetString() directly,
```

Comment:
@shenli 
For example, `Bit` will be regarded as ClassString in runtime refer to [MySQL](https://github.com/mysql/mysql-server/blob/5.7/sql/field.cc#L1069),
we store Bit in [Datum.i](https://github.com/pingcap/tidb/blob/master/util/types/datum.go#L218), while Datum.GetString() gets the value from [Datum.b](https://github.com/pingcap/tidb/blob/master/util/types/datum.go#L1344) directly.